README.md

```diff
@@ -303,6 +303,7 @@ _See also: [awesome-selfhosted/Database Management](https://github.com/awesome-s
 - [Diagrams.net](https://app.diagrams.net/) - A.K.A. [Draw.io](https://app.diagrams.net/). Easy to use Diagram UI with a plethora of templates.
 - [DrawThe.Net](http://go.drawthe.net/) - Javascript tool that uses a YAML-formatted input to programmatically create large, complex, and visually solid diagrams.
 - [Mermaid](https://mermaid-js.github.io/mermaid-live-editor/) - Javascript module with a unique, easy, shorthand syntax. Integrates into several other tools like Grafana.
+- [Structurizr](https://www.structurizr.com/) - Architecture diagramming/modelling UI for developing and collaborating on diagrams-as-code aligned to the C4 model for architecture diagrams. Facilitates workflows for creating multiple software architecture diagrams from a single model.
```

Comment:
Missing several required fields (source code URL, license, language/platform), please refer to the Pull Request template and follow the recommended format.

Other entries in this section are also missing the required fields, because I did not have time to work on https://github.com/awesome-foss/awesome-sysadmin/issues/416 on this section yet.